modulefiles/modulefile.ProdGSI.orion

```diff
@@ -9,36 +15,37 @@ set COMP_MPI mpiifort
 set C_COMP icc
 set C_COMP_MP mpcc
 
-# Known conflicts
+setenv CRTM_FIX /apps/contrib/NCEPLIBS/orion/fix/crtm_v2.3.0
 
+# Known conflicts
```

Comment:
are there any?
If not, please remove the comment line

Reply:
Conflict line removed.